docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-portal-product-offer-management-feature-walkthrough.md

```diff
@@ -12,11 +12,16 @@ To learn more about the feature and to find out how end users use it, see [Marke
 
 {% endinfo_block %}
 
+## Module dependency graph
 
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/c7d38902-eec0-417d-94ce-31d1baf9599d.png?utm_medium=live&utm_source=custom)
```

Comment:
Please add the main modules that `ProductOfferMerchantPortalGui` depends on. (Please use white color for them)